Gemfile

```diff
@@ -56,8 +56,8 @@ group :development, :test do
   gem 'database_cleaner'
 end
 
-gem 'blacklight', '~> 5.12.1'
-gem 'geoblacklight', '~> 0.10.2'
+gem 'blacklight', '~> 5.12.0'
```

Comment:
It sure does: https://github.com/geoblacklight/geoblacklight/blob/59c2c4e0b308bfc7d753858cba589f1de53a1596/geoblacklight.gemspec#L21